docs/install/v2_1_3_full.yaml

```diff
@@ -485,6 +485,8 @@ spec:
         - args:
             - --cluster-name=your-cluster-name
             - --ingress-class=alb
+            - --aws-region=eks-vpc-region
```

Comment:
These are auto-discovered if IMDS is enabled. We didn't put in the sample manifest so as to avoid confusion. These should be added manually during installation.

IMO it is better if we mention this in the intallation guide.